pkg/operator/ceph/csi/spec.go

```diff
@@ -605,15 +605,24 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string, ownerRef *
 		},
 	}
 	csidrivers := clientset.StorageV1beta1().CSIDrivers()
-	k8sutil.SetOwnerRef(&csiDriver.ObjectMeta, ownerRef)
-	_, err := csidrivers.Create(csiDriver)
+	driver, err := csidrivers.Get(name, metav1.GetOptions{})
```

Comment:
Instead of these changes, can we just cherry-pick [this commit](https://github.com/rook/rook/commit/743a41cfa112643fb024fd6d148c6208cba5988d#diff-81a827d592f5889c0d4dd6a8cbf80218852d57f5ffa6f3babd6f1944fcd710af) that was already made upstream?

Reply:
Oh, now I remember that we made a different change in master with https://github.com/rook/rook/pull/6228, and you're just integrating that.

Reply:
Yes correct @travisn, This is a cherry-pick from the master branch